.travis.yml

```diff
@@ -0,0 +1,21 @@
+dist: trusty
+services:
+- docker
+language: go
+go:
+- 1.12.x
+install:
+- bash scripts/gogetcookie.sh
+script:
+- make test
+branches:
+  only:
```

Comment:
@pdecat should not run these tests on every branches ?

Reply:
I questioned myself on this one but noticed the tests are already running for PRs so left it that way.

https://travis-ci.org/claranet/terraform-provider-zabbix/builds/623160813#L817

It is probably be useful when committing to branches other than master outside of a PR.